demos/python_demos/asl_recognition_demo/asl_recognition_demo/video_stream.py

```diff
@@ -0,0 +1,177 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import time
+import signal
+from os.path import exists
+from multiprocessing import Process, Value, Array
+
+import cv2
+import numpy as np
+
+
+class VideoStream:
+    """ This class returns constant framerate frames from the input stream. """
+
+    def __init__(self, input_source, trg_fps, batch_size):
+        """Conctructor"""
```

Comment:
There's a typo, but also this is just a useless comment.

Reply:
Fixed.